tamr_unify_client/models/project/attribute_mapping.py

```diff
@@ -0,0 +1,77 @@
+from tamr_unify_client.models.base_resource import BaseResource
```

Comment:
This probably makes more sense to be in its own `attribute_mapping` directory or the `attribute` directory if not, rather than project